### ISSUE TITLE
Numpy API missmatch with v1.3.3 wheel 

### ISSUE THREAD

**Issue by montefra** [astropy/astropy#6133]
A collegue of mine and I are having the following error on our software when importing ``astropy.tables`` when using the v1.3.3 wheel.

```
    from astropy.table import Table, vstack
.tox/py36/lib/python3.6/site-packages/astropy/table/__init__.py:43: in <module>
    from .column import Column, MaskedColumn
.tox/py36/lib/python3.6/site-packages/astropy/table/column.py:20: in <module>
    from . import groups
.tox/py36/lib/python3.6/site-packages/astropy/table/groups.py:11: in <module>
    from .index import get_index
.tox/py36/lib/python3.6/site-packages/astropy/table/index.py:42: in <module>
    from ..time import Time
.tox/py36/lib/python3.6/site-packages/astropy/time/__init__.py:2: in <module>
    from .formats import *
.tox/py36/lib/python3.6/site-packages/astropy/time/formats.py:15: in <module>
    from .. import _erfa as erfa
.tox/py36/lib/python3.6/site-packages/astropy/_erfa/__init__.py:4: in <module>
    from .core import *
.tox/py36/lib/python3.6/site-packages/astropy/_erfa/core.py:43: in <module>
    from . import _core
E   ImportError: numpy.core.multiarray failed to import
---------------------------------------------------------------------------- Captured stderr -----------------------------------------------------------------------------
RuntimeError: module compiled against API version 0xb but this version of numpy is 0xa
```

If I use v1.3.3 tar file (with ``pip install --no-binary :all: astropy``) or other versions of astropy (I have tested v.1.3.2 and v1.2 the wheels) everything is fine.

If you need more info let me know.

**Comment by pllim** [comment]
Looks like a duplicate of #5939

**Comment by montefra** [comment]
It's a similar issue, however I'm not sure it is a duplicate, as it appears only with the v1.3.3 wheels on a newly created virtualenv.
I have been using the tox/astropy/numpy combo for the past couple of years and I have never seen this.

I forgot to mention: it's on a Linux machine.

**Comment by pllim** [comment]
What is the installation path of Numpy in that env?

**Comment by montefra** [comment]
Numpy is installed in the same virtual environemnt.

And: I just realized that pypi doesn't have 1.3.3 wheels, so I wrongly blamed the wheels. Apologies.

So I did try a few more cases (e.g. ``pip install --no-use-cache astropy``) and all worked fine.
As last I removed ~/.cache/pip and I cannot reproduce the issue anymore.

Now I'm even more puzzled than before.
Could it be some weird interference between issue #6130 and the pip cache mechanism that made pip use the wrong version of astropy and/or numpy?

**Comment by pllim** [comment]
Could be. If you think this has been resolved on your end, please close the issue. Thanks for the report!

**Comment by montefra** [comment]
I'm running ``tox`` after updating it and clearing the pip cache. I'll let you know how it goes (probably tomorrow).

**Comment by montefra** [comment]
With ``tox`` I still see the problem with v1.3.3. 

The tox logs show:

    Running setup.py bdist_wheel for astropy: started
    Running setup.py bdist_wheel for astropy: still running...
    Running setup.py bdist_wheel for astropy: still running...
    Running setup.py bdist_wheel for astropy: finished with status 'done'

So for some reason ``tox`` is building wheels, get the ok and use them. After I have the wheels chached all other ``pip install`` will use those broken wheels.
This doesn't happen for v1.3.2, as it already comes with wheels.

ps: For the moment in my case I am disabling v1.3.3 in the requirements

**Comment by timstaley** [comment]
Just to add, I'm seeing this issue in a tox / virtualenv / travis setup also. The travis-cron-builds suddenly started failing for a bunch of my repos, tracing it back from this issue I see it clearly matches the 1.3.3 release:
1.3.2, Worked:
https://travis-ci.org/timstaley/voevent-parse/builds/237260477
1.3.3, Failed:
https://travis-ci.org/timstaley/voevent-parse/builds/237628466
What's weird is that it runs OK on my local system.
I had a hunch that it might be finding the system-version of Numpy rather than performing a local install (hence the version mismatch), so I added a manual 'install Numpy' command which seems to help:
https://travis-ci.org/timstaley/voevent-parse/builds/240560048
... though I'm still seeing issues with my sphinx setup.

**Comment by timstaley** [comment]
And... I seem to have a working work-around:
https://github.com/timstaley/voevent-parse/commit/7e643523ef8e3bb46552fa32e5c5cecd6ed1dd71
https://travis-ci.org/timstaley/voevent-parse/builds/240566799

**Comment by astrofrog** [comment]
@matthew-brett - any idea what this could be caused by?

cc also @bsipocz

**Comment by matthew-brett** [comment]
It sounds like this isn't anything to do with the wheels, but in fact only happens when you don't use the wheels.  I suppose what must be happening is that astropy is building against a very recent numpy, and then somehow numpy is being downgraded, causing the error.  But I don't know what would cause that.

One take-home might be to make sure the wheels go up at the same time or before the source release.

I've set off a build for 1.3.3 now in the hope that uploading the result will fix this.

**Comment by bsipocz** [comment]
If it should be done before or with the source release, could you add the necessary steps to the releasing docs? 
http://docs.astropy.org/en/latest/development/releasing.html

**Comment by matthew-brett** [comment]
I made a pull request with the wheel building procedure: https://github.com/astropy/astropy/pull/6241

OK to upload the built wheels for 1.3.3?

**Comment by bsipocz** [comment]
@matthew-brett - Thanks for the PR! Also, I think it's OK to have the wheels uploaded for 1.3.3

**Comment by bsipocz** [comment]
We revised the release procedure to include the builds of the wheels, and that should have addressed this issue. Please reopen if you think otherwise.
